addons/io_scene_gltf2/__init__.py

```diff
@@ -258,19 +258,19 @@ class ExportGLTF2_Base:
     )
 
     export_lights = BoolProperty(
-        name='Export KHR_lights_punctual',
+        name='Export punctual lights',
         description='',
```

Comment:
I'm fine with this rename.

Question, does this description show up on the UI?  Is it worth reporting the extension name here?

Reply:
The descriptions do show up in the UI, yes. Great idea. I'll do a pass on this and fill in all of the descriptions — seems like they may be helpful even for non-extension options.